tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -473,8 +473,8 @@ def fit(self,
                 Hyperparameter tuning strategy and kwargs (for example, how many HPO trials to run).
                 If None, then hyperparameter tuning will not be performed.
                 Valid preset values:
-                    'auto': Uses the 'local_sequential_auto' preset.
-                    'local_sequential_auto': Performs sequential local search via bayesopt
+                    'auto': Uses the 'local' preset.
+                    'local': Performs sequential local search via bayesopt
```

Comment:
For user, it will not be clear how "sequential local search via bayesopt" is different than the 'bayesopt' option listed below.

Reply:
this is changed to all local; auto==bayesopt now